runtime/lua/vim/lsp/util.lua

```diff
@@ -804,9 +804,11 @@ function M.convert_signature_help_to_markdown_lines(signature_help, ft, triggers
       active_offset[2] = active_offset[2] + #contents[1]
     end
 
-    active_hl = {}
-    list_extend(active_hl, get_pos_from_offset(active_offset[1], contents) or {})
-    list_extend(active_hl, get_pos_from_offset(active_offset[2], contents) or {})
+    local astart = get_pos_from_offset(active_offset[1], contents)
+    local aend = get_pos_from_offset(active_offset[2], contents)
+    if astart and aend then
```

Comment:
Super nit but I find `astart` and `aend` to be not very useful variable names.

Reply:
The `a` is for active_offset. I would have used `start` and `end` but `end` is a keyword so I added a single letter prefix. I could go with `s` and `e`. Their scope is very small, so I don't think longer names are justified.

Reply:
Can we at least use `a_`?